licenseheader.txt

```diff
@@ -0,0 +1,13 @@
+/* 
```

Comment:
Note: you can simply add another commit to this pull request, no need to create a new one